eng/pipelines/common/xplat-setup.yml

```diff
@@ -181,7 +181,7 @@ jobs:
         # Official Build Windows Pool
         ${{ if and(or(eq(parameters.osGroup, 'windows'), eq(parameters.jobParameters.hostedOs, 'windows')), ne(variables['System.TeamProject'], 'public')) }}:
           name: $(DncEngInternalBuildPool)
-          demands: ImageOverride -equals windows.vs2022.amd64
+          demands: ImageOverride -equals 1es-windows-2022 # ???
```

Comment:
so this needs fixing yet.

Reply:
looks like it's not - works without this change just fine, reverted